lib/ansible/modules/packaging/language/maven_artifact.py

```diff
@@ -41,59 +41,63 @@
 options:
     group_id:
         description:
-            - The Maven groupId coordinate
+            - The maven groupId coordinate.
         required: true
     artifact_id:
         description:
-            - The maven artifactId coordinate
+            - The maven artifactId coordinate.
         required: true
     version:
         description:
-            - The maven version coordinate
+            - The maven version coordinate. Special value "release" is used to specify the latest available release.
         required: false
         default: latest
     classifier:
         description:
-            - The maven classifier coordinate
+            - The maven classifier coordinate.
         required: false
         default: null
     extension:
         description:
-            - The maven type/extension coordinate
+            - The maven type/extension coordinate.
         required: false
         default: jar
-    repository_url:
+    url_repository:
+        version_added: "2.3"
+        aliases: [ repository_url, repository ]
         description:
-            - The URL of the Maven Repository to download from.
+            - The URL of the maven Repository to download from.
             - Use s3://... if the repository is hosted on Amazon S3, added in version 2.2.
         required: false
         default: http://repo1.maven.org/maven2
-    username:
+    url_username:
+        version_added: "2.3"
+        aliases: [ username, user, uname, aws_secret_key ]
         description:
-            - The username to authenticate as to the Maven Repository. Use AWS secret key of the repository is hosted on S3
+            - The username to authenticate as to the maven Repository, Use AWS secret key of the repository is hosted on S3.
         required: false
         default: null
-        aliases: [ "aws_secret_key" ]
-    password:
+    url_password:
+        version_added: "2.3"
+        aliases: [ password, pwd, passwd, aws_secret_access_key ]
```

Comment:
Are these common aliases with maven?
Generally with modules we just use `user:` and `password`